include/vinecopulib/vinecop/implementation/class.ipp

```diff
@@ -1157,6 +1295,45 @@ Vinecop::truncate(size_t trunc_lvl)
   }
 }
 
+//! set all variable types to continuous.
+//! the function can be const, because var_types_ is mutable.
+inline void
+Vinecop::set_continuous_var_types() const
+{
+  var_types_ = std::vector<std::string>(d_);
+  for (auto& t : var_types_)
+    t = "c";
```

Comment:
Don't you need to adjust the types for all pairs too?

Reply:
Indeed